numba/np/arraymath.py

```diff
@@ -4040,13 +4041,24 @@ def histogram_impl(a, bins=10, range=None):
 
 iinfo = namedtuple('iinfo', _iinfo_supported)
 
+# Suppress deprecation warnings while keeping support MachAr
+with warnings.catch_warnings(record=True):
```

Comment:
```suggestion
with warnings.catch_warnings():
```
No need to remember the warnings